modules/users/users.controller.js

```diff
@@ -0,0 +1,59 @@
+const { User } = require("./user.schema");
+
+const getUsers = async (req, res) => {
+  const users = await User.find();
+  return res.json({ data: users });
+};
+
+const getUserById = async (req, res) => {
+  const user = await User.findOne({ _id: req.params.id });
```

Comment:
Czy to zapytanie jest zabezpieczone gdyby tu Wystąpił błąd? To znaczy czy jest w bloku try-catch gdzieś powyżej?

Reply:
Dodałam try catch